src/emulator/gui/src/game_selector.cpp

```diff
@@ -179,22 +190,105 @@ void draw_game_selector(GuiState &gui, HostState &host) {
                     }
                     if (ImGui::IsMouseClicked(0))
                         selected[0] = true;
+                    if (gui.title_id != game.title_id)
```

Comment:
I said I wouldn't complain about checking for strings but I want to at least mention that you're still doing a lot of checking. Checking and comparing every letter in both strings 60 times a second vs just copying the contents over 60 times a second with less bloat.

Please consider my plea!

Reply:
Sorry but i no change this, check value is safe tandis just normal ^^